app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,46 @@
+class QuestionsController < ApplicationController
+  def index
+    @questions = Question.all
+  end    
+
+  def show
+    @question = Question.find(params[:id])
+    
+    if @question.resolved 
```

Comment:
This is really view logic (IE what should be displayed to the user), so I would put this directly in the view.